sample-compose/buildSrc/src/main/java/Versions.kt

```diff
@@ -1,7 +1,7 @@
 object Versions {
     const val BUILD_GRADLE_VERSION = "4.2.1"
 
-    const val ANDROID_COMPILE_SDK_VERSION = 30
+    const val ANDROID_COMPILE_SDK_VERSION = 31
```

Comment:
@Tuubz @chornerman  Shall we align the version with the main template too? 🤔 Or we can update it separately on each sample?

Reply:
@lydiasama if we align with the main template, does that mean we'll need to update both samples too each time we change something? 🤔

Reply:
Yeah @ryan-conway  That's the idea 👍🏻  I think it's good to clarify this time to be aligned. 🙌🏻

Reply:
Resolved: https://github.com/nimblehq/android-templates/discussions/279